apps/ganache/staging/helm/ganache-helm-values.yml

```diff
@@ -0,0 +1,209 @@
+# -- Overrides the chart's name
+nameOverride: ""
+
+# -- Overrides the chart's computed fullname
+fullnameOverride: ""
+
+image:
+  # -- Ganache container image repository
+  repository: trufflesuite/ganache-cli
+  # -- Ganache container image tag
+  tag: v6.12.2
+  # -- Ganache container pull policy
+  pullPolicy: IfNotPresent
+
+# -- Extra args for the ganache container
+extraArgs: []
+  #- --accounts=10
+  #- --mnemonic="$(MNEMONIC)"
+
+# -- Command replacement for the ganache container
+customCommand: [] # Only change this if you need to change the default command
+
+ingress:
+  # -- Ingress resource for the HTTP API
+  enabled: false
+  # -- Annotations for Ingress
+  annotations: 
+    cert-manager.io/cluster-issuer: letsencrypt-staging
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/tls-acme: "true"
+  # -- Ingress host
+  hosts:
+    - host: ganache.omni.cedille.club
+      paths: 
+        - path: /
+          pathType: Prefix
+          backend:
+            service:
+              name: my-ganache
+              port:
+                number: 8545
+  # -- Ingress TLS
+  tls: []
+    - secretName: ganache-tls
+      hosts:
+        - ganache.omni.cedille.club
+
+service:
+  # -- Service type
+  type: ClusterIP
```

Comment:
Is this the right type of service ?

Reply:
Yes, by default and we want this

Reply:
By default it also create a headless service